src/data/graph-data.js

```diff
@@ -79,6 +81,49 @@ async function getLinkedIssues(
   });
 }
 
+export async function GetAllOrganizations(
+  endpointUrl,
+  zenhubApiKey,
+  signal
+) {
+  const gqlQuery = createGqlQuery(endpointUrl, zenhubApiKey, signal);
+
+  const {
+    viewer: {
+      zenhubOrganizations: {
+        nodes: organizations,
+      }
+    }
+  } = await gqlQuery(GET_ALL_ORGANIZATIONS, "GetAllOrganizations", {});
+
+  return organizations.map( ( organization ) => ( {
+    id: organization.id,
+    name: organization.name,
+    workspaces: organization.workspaces.nodes.map( ( workspace ) => ( { id: workspace.id, name: workspace.name } ) ),
+  } ) );
+}
+
+export async function getAllEpics(
+  workspaceId,
+  endpointUrl,
+  zenhubApiKey,
+  signal
+) {
+  const gqlQuery = createGqlQuery(endpointUrl, zenhubApiKey, signal);
+
+  const {
+    workspace: {
+      epics: {
+        nodes: epics,
+      }
+    }
+  } = await gqlQuery(GET_ALL_EPICS, "GetAllEpics", {
+    workspaceId,
+  });
+
+  return epics.map( ( epic ) => epic.issue );
```

Comment:
As above, although not so compelling:

```suggestion
  return epics.map( ( { issue } ) => issue );
```